web/source/osk/visualKeyboard.ts

```diff
@@ -1054,10 +1049,13 @@ namespace com.keyman.osk {
 
       // Stop repeat if no longer on BKSP key
       if(key1 && (typeof key1.id == 'string') && (key1.id.indexOf('BKSP') < 0)) {
```

Comment:
I know this isn't part of this PR, but why are we using `indexof('BKSP')` instead `== 'K_BKSP'`?

Reply:
Because it's actually 'default-K_BKSP`, `shift-K_BKSP`, or `<insert layer id here>-K_BKSP`.  Each layer's BKSP key has its own unique ID because of this scheme.

Reply:
Okay. Long term it'd be better to test `-K_BKSP` to avoid false positives with `T_MABKSPAN` or other weird key identifiers that devs may use. :)